<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowScrenn</class>
 <widget class="QWidget" name="ShowScrenn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(0, 0, 0,0);
</string>
  </property>
  <widget class="QQuickWidget" name="quickWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>150</height>
    </size>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>36</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="Zoom_Out_Button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(70,70, 70, 55); /* Butonun arka plan rengi, şeffaf koyu mavi */
    border-radius: 20px; /* Yuvarlak köşeler */
    text-align: center; /* Metni ortala */
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::ZoomOut"/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Zoom_In_Button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(70,70, 70, 55);  /* Butonun arka plan rengi, şeffaf koyu mavi */
    border-radius: 20px; /* Yuvarlak köşeler */
    text-align: center; /* Metni ortala */
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::ZoomIn"/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Default_Location_Button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string comment="Go Home Location" extracomment="Go Home Location"/>
      </property>
      <property name="statusTip">
       <string comment="Go Home Location" extracomment="Go Home Location"/>
      </property>
      <property name="whatsThis">
       <string comment="Go Home Location" extracomment="Go Home Location"/>
      </property>
      <property name="accessibleName">
       <string comment="Go Home Location" extracomment="Go Home Location"/>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color:  rgba(70,70, 70, 55); /* Butonun arka plan rengi, şeffaf koyu mavi */
    border-radius: 20px; /* Yuvarlak köşeler */
    text-align: center; /* Metni ortala */
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::GoHome"/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
